server/routes/request.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  makeRequest,
+  getRequest,
+  editRequest,
+} = require("../controllers/request");
+
+router.route("/").post(protect, makeRequest);
+
+router.route("/").get(protect, getRequest);
+
+router.route("/").patch(protect, editRequest);
```

Comment:
If you choose to add request id to you request params, you can now make this one a dynamic route like this:
`router.route("/:id").patch(protect, editRequest);`

Reply:
Changed implemented as noted above.